tests/wfqueueworker.ts

```diff
@@ -68,14 +70,12 @@ async function main() {
 
   // Wait to dequeue as many tasks as we can locally
   for (let i = 0; i < localConcurrencyLimit; i++) {
-    await InterProcessWorkflowTask.start_event.wait();
-    InterProcessWorkflowTask.start_event.clear();
```

Comment:
I also notice that this was subject to a race: if all tasks are dequeued before the first (or, more likely, second) `wait`, the `set()` would all have resolved the same promise and the subsequent `wait()`, well, not wait on anything.